tests/acceptance/test_files/chaos_and_query.bats

```diff
@@ -2,23 +2,39 @@ load "$LIB_BATS_ASSERT/load.bash"
 load "$LIB_BATS_SUPPORT/load.bash"
 
 @test "select * from chaos.chaos_high_row_count order by column_0" {
-  run steampipe query --output json  "select * from chaos.chaos_high_row_count order by column_0 limit 10"
-  assert_equal "$output" "$(cat $TEST_DATA_DIR/expected_1.json)"
+  steampipe query --output json  "select * from chaos.chaos_high_row_count order by column_0 limit 10" > output.json
+  run jd -f patch "$TEST_DATA_DIR/expected_1.json" output.json
+  diff=$($FILE_PATH/json_patch.sh $output)
+  rm output.json
+
+  assert_equal $diff ""
 }
 
 @test "select id, string_column, json_column, boolean_column from chaos.chaos_all_column_types where id='0'" {
-  run steampipe query --output json  "select id, string_column, json_column, boolean_column from chaos.chaos_all_column_types where id='0'"
-  assert_equal "$output" "$(cat $TEST_DATA_DIR/expected_2.json)"
+  steampipe query --output json  "select id, string_column, json_column, boolean_column from chaos.chaos_all_column_types where id='0'" > output.json
+  run jd -f patch "$TEST_DATA_DIR/expected_2.json" output.json
+  diff=$($FILE_PATH/json_patch.sh $output)
+  rm output.json
+
+  assert_equal $diff ""
```

Comment:
Similarly here:
```suggestion
  run jd "$TEST_DATA_DIR/expected_2.json" output.json
  echo $output
  assert_success
  rm -f output.json
```

Similarly, update all other instances(you have modified) in this file